cmd/ak/cmd/export/export.go

```diff
@@ -0,0 +1,143 @@
+package export
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+var Cmd = common.StandardCommand(&cobra.Command{
+	Use:     "export <project name or ID>",
+	Short:   "Export project",
+	Aliases: []string{"ex"},
+	Args:    cobra.ExactArgs(1),
+
+	RunE: export,
+})
+
+func export(cmd *cobra.Command, args []string) error {
+	r := resolver.Resolver{Client: common.Client()}
+	ctx, cancel := common.LimitedContext()
+	defer cancel()
+
+	name := args[0]
+	prj, _, err := r.ProjectNameOrID(ctx, name)
+	if err != nil {
+		return err
+	}
+
+	if !prj.IsValid() {
+		return fmt.Errorf("project %q not found", name)
+	}
+
+	tf := sdkservices.ListTriggersFilter{
+		ProjectID: prj.ID(),
+	}
+	triggers, err := r.Client.Triggers().List(ctx, tf)
+	if err != nil {
+		return err
+	}
+
+	// We print to buffer until the end so in case of errors we won't print anything but the error
+	var buf bytes.Buffer
+	fmt.Fprintln(&buf, "version: v1")
+	fmt.Fprintln(&buf)
+	fmt.Fprintln(&buf, "project:")
```

Comment:
isn't a better way to use the `manifest` package and just serialize it?

Reply:
Yes it is, forgot about that - will change.

Reply:
Done